File: epargne/DTO/TransactionEpargneDto.cs
using System;

namespace Epargne.DTO
{
    public class TransactionEpargneDto
    {
        public int IdTransaction { get; set; }
        public int IdCompte { get; set; }
        public DateOnly DateTransaction { get; set; }
        public string Libelle { get; set; }
        public decimal Montant { get; set; }
        public string Sens { get; set; } // debit ou credit
    }
}

File: server-ejb/src/main/java/com/example/dto/TransactionEpargneDto.java
package com.example.dto;

import java.io.Serializable;
import java.math.BigDecimal;
import java.time.LocalDate;

import lombok.Data;

@Data
public class TransactionEpargneDto implements Serializable {

    private int idTransaction;
    private int idCompte;
    private LocalDate dateTransaction;
    private String libelle;
    private BigDecimal montant;
    private String sens; // "debit" ou "credit"
}
